Add blog post about xxxlutz.md (#154)
kyma-project/website: src/blog-posts/9-xxxlutz-videoo.md

---
path: "/blog/xxxlutz-video"
date: "2018-10-12"
author: "Annegret Stoye, Product Manager @Kyma"
tags: ["kyma", "opensource", "innovation", "technology"]
title: "Fueling Innovation with Customer Feedback"
---

In the early stages of developing Kyma, the focus of Kyma to extend and customize enterprise applications using serverless computing or microservice architecture and the fact that Kyma uses existing open source technologies raised interest among companies.

Innovative companies are constantly seeking to identify and experience new technology solutions to assess potential benefits for the company. Receiving feedback from companies early on is very valuable for the Kyma team to make improvements before general availability.

> Before going Open Source with Project Kyma, it was pre-launched to a selected group of companies. The XXXLutz KG was one of the companies to get early access to Kyma and test it.

Reviewer: ```suggestion
Before going open source, Kyma was prelaunched to a selected group of companies. One of them was XXXLutz KG, one of the world’s largest furniture retailers. They received privileged early access to Kyma and were ecouraged to test it. If you are interested in the outcome, watch [this](https://youtu.be/rxeE05NopZI) video where Christian Renner, the Chief Technology Officer of the XXXLutz Group, explains why they wanted to test Kyma and what their experience with the project was like.

```

> Christian Renner, Chief Technology Officer XXXLutz Gruppe, one of the world&#39;s largest furniture retailers, shared why they wanted to test Kyma and what their experience was like.

Reviewer: ```suggestion

```

> Getting feedback and contributions from the Open Source community remains our focus moving forward. We invite you to start your very own Kyma journey!

Reviewer: ```suggestion
Getting feedback and contributions from the open-source community remains our focus moving forward. Feel invited to start your very own Kyma journey:
```

> [Install](https://kyma-project.io/docs/root/kyma#installation-installation) and test Kyma, explore the examples on [GitHub](https://github.com/kyma-project), provide your feedback via [Slack](https://join.slack.com/t/kyma-community/shared_invite/enQtNDAwNzE4Mjk2NDE3LTJhOTlmZjM5YzkwNmEzNmY3ZjE2MTU2OTMxOGE4ZDM0MmU4ZWRkZGJiODgzNmRmMTYxMDYwNjZiMDAwMTA2OWM) and join the Kyma community on [Twitter](https://twitter.com/kymaproject) and [LinkedIn](https://www.linkedin.com/authwall?trk=ripf&amp;trkInfo=AQFL2ZrSEnYW9gAAAWd9pfdAKay9_IjJ5fE58rtE5CvgTVY1LQicYsSOd52ICurd4z7RsU4dhbqiMrP5IKO70z1OmhxEUJyreAQsmDllpNYlAY334UVCr-5T7_cvzf15RfP5LtA=&amp;originalReferer=&amp;sessionRedirect=https%3A%2F%2Fwww.linkedin.com%2Fcompany%2Fkyma-project%2F) for the latest news.

Reviewer: ```suggestion
- [Install](https://kyma-project.io/docs/root/kyma#installation-installation) Kyma and test it.
- Explore the examples on [GitHub](https://github.com/kyma-project).
- Share your feedback on [Slack](https://join.slack.com/t/kyma-community/shared_invite/enQtNDAwNzE4Mjk2NDE3LTJhOTlmZjM5YzkwNmEzNmY3ZjE2MTU2OTMxOGE4ZDM0MmU4ZWRkZGJiODgzNmRmMTYxMDYwNjZiMDAwMTA2OWM). 
- Join the Kyma community on [Twitter](https://twitter.com/kymaproject) and [LinkedIn](https://www.linkedin.com/authwall?trk=ripf&amp;trkInfo=AQFL2ZrSEnYW9gAAAWd9pfdAKay9_IjJ5fE58rtE5CvgTVY1LQicYsSOd52ICurd4z7RsU4dhbqiMrP5IKO70z1OmhxEUJyreAQsmDllpNYlAY334UVCr-5T7_cvzf15RfP5LtA=&amp;originalReferer=&amp;sessionRedirect=https%3A%2F%2Fwww.linkedin.com%2Fcompany%2Fkyma-project%2F) for the latest news.
```

> We are looking forward to engaging with you!

Reviewer: ```suggestion
We are looking forward to your feedback and engagement!
```

---
path: "/blog/xxxlutz-video"
date: "2018-10-12"
author: "Annegret Stoye, Product Manager @Kyma"
tags: ["kyma", "opensource", "innovation", "technology"]
title: "Fuel innovation with customer feedback"
---

From the early stages of its development, Kyma has raised interest among other companies. The main reasons for that were Kyma's existing open-source technologies that consititute its cornerstone, and the fact that it allows companies to extend and customize their enterprise applications using serverless computing or microservice architecture.

> Innovative companies are constantly seeking ways to identify and experience new technology solutions that can provide potential benefits to their businesses. Receiving feedback from such companies at the early stage was very valuable. As a result, the Kyma team could make the necessary improvements to the product and launch its refined version as an open-source project. 

Reviewer: ```suggestion
Innovative companies are constantly seeking ways to identify and experience new technology solutions that can provide potential benefits to their businesses. Receiving feedback from such companies at the early stage was very valuable to the Kyma team. As a result, we could make the necessary improvements to the product and launch its refined version as an open-source project. 
```

> Before going open source, Kyma was prelaunched to a selected group of companies. One of them was XXXLutz KG, the world’s top furniture retailer. The company representatives received privileged early access to Kyma and were encouraged to test it. If you are interested in the outcome, watch [this](https://youtu.be/rxeE05NopZI) video where Christian Renner, the Chief Technology Officer of the XXXLutz Group, explains why they wanted to test Kyma and what their experience with the project was like.

Reviewer: ```suggestion
Before going open-source, Kyma was prelaunched to a selected group of companies. One of them was XXXLutz KG, the world’s top furniture retailer. The company representatives received privileged early access to Kyma and were encouraged to test it. If you are interested in the outcome, watch [this](https://youtu.be/rxeE05NopZI) video where Christian Renner, the Chief Technology Officer of the XXXLutz Group, explains why they wanted to test Kyma and what their experience with the project was like.
```
